tests/storage/business/test_xpansion_manager.py

```diff
@@ -289,7 +289,7 @@ def test_update_xpansion_settings(tmp_path: Path) -> None:
         "master": Master.INTEGER,
         "yearly-weights": None,
         "additional-constraints": None,
-        "relaxed-optimality-gap": "1.2%",
+        "relaxed_optimality_gap": "1.2%",
```

Comment:
Curious! This value should be a float without dimension, but not a percentage.

Reply:
Indeed, but we authorize str values for the R version :/